src/AbcSimplex.cpp

```diff
@@ -31,11 +37,11 @@
 #include "CoinAbcHelperFunctions.hpp"
 #include "CoinModel.hpp"
 #include "CoinLpIO.hpp"
-#include <cfloat>
 
-#include <string>
-#include <stdio.h>
-#include <iostream>
+#ifdef COINUTILS_HAS_GLPK
+#include "glpk.h"
+#endif
```

Comment:
Similar as in Osi: There should be a check in Clp for glpk.h to ensure that its header is found. Just because the CoinUtils lib was build with Glpk as dependency does not mean that the necessary and right `-I` flags are also available for Clp.